openshift_gcp.py

```diff
@@ -56,12 +56,12 @@ def set_openshift_gcp_project_from_google_creds(self):
 
     def set_openshift_gcp_project_from_metadata(self):
         try:
-            r = requests.get(
-                'http://metadata.google.internal/computeMetadata/v1/project/project-id',
-                headers = { "Metadata-Flavor": "Google" }
-            )
-            r.raise_for_status()
-            self.ocpinv().set_dynamic_cluster_var('openshift_gcp_project', r.text)
+            url = 'http://metadata.google.internal/computeMetadata/v1/project/project-id'
+            headers = { "Metadata-Flavor": "Google" }
+            req = urllib2.Request(url, None, headers)
+            resp = urllib2.urlopen(req)
+            p = resp.read()
+            self.ocpinv().set_dynamic_cluster_var('openshift_gcp_project', p.text)
         except:
             traceback.print_exec()
```

Comment:
I spotted an error here too, though it was old code, can you change this from `print_exec` to `print_exc` in your PR?

Reply:
No problem, done